js/ui/shellDBus.js

```diff
@@ -610,6 +610,12 @@ var HackableAppsManager = class {
             paths.push(session.hackableApp.objectPath);
         return paths;
     }
+
+    PulseFtH(appid, enable) {
+        let session = this._codeViewManager.sessions.find((s) => s.appId === appid);
```

Comment:
Maybe it would be better to add this method on the HackableApp class?

Reply:
Yes, you're right, I'll move to there.

Reply:
Done.